labs/crazyegg-integration/README.md

```diff
@@ -0,0 +1,26 @@
+# Crazy Egg Integration
+
+Crazy Egg integration allows you to see heatmaps with Optimizely data. For more information on how to use this integration, please take a look at this Knowledge Base article: https://help.optimizely.com/Integrate_Optimizely_with_Crazyegg
```

Comment:
This web page looks like it goes to a 404 not found: https://help.optimizely.com/Integrate_Optimizely_with_Crazyegg

Reply:
Also, it I'd encourage us to specify what product this integration is for: Optimizely Web? Optimizely Full Stack? Some other Optimizely products?

Reply:
@asaschachar do we need to point to a help article? We may have been looking at some older instructions that linked to a help article on Optimizely's site.